src/porepy/numerics/fv/biot.py

```diff
@@ -843,7 +850,26 @@ def assemble_rhs(self, g, data):
             np.ndarray: Zero right hand side vector with representation of boundary
                 conditions.
         """
-        return np.zeros(self.ndof(g))
+        parameter_dictionary = data[pp.PARAMETERS][self.keyword]
+        matrix_dictionary = data[pp.DISCRETIZATION_MATRICES][self.keyword]
+
+        # For IE and constant BCs, the boundary part cancels, as the contribution from
+        # successive timesteps (n and n+1) appear on the rhs with opposite signs. For
+        # transient BCs, use the below with the appropriate version of d_bound_i.
+        d_bound_1 = parameter_dictionary["bc_values"]
```

Comment:
Is this correct? Will not d_bound_1 == d_bound_0

Reply:
I belive it is correct. If you have constant boundary conditions  d_bound_1 == d_bound_0, but this is not true for time dependent boundary conditions. Time dependent boundary conditions are not supported yet, but Ivar and I agreed it was best to still write the equation down to make it explicit

Reply:
I should have read the comments more carefully then. Good.